pycm/pycm_param.py

```diff
@@ -39,6 +39,10 @@
 
 COMPARE_RESULT_WARNING = "Confusion matrices are too close and the best one can not be recognized."
 
+AVERAGE_WEIGHT_ERROR = "The weight type must be dictionary and also must be set for all classes."
```

Comment:
I think it's better to be written as follows:

"The weight type must be dictionary and also must be specified for all of the classes."